README.md

```diff
@@ -52,3 +52,7 @@ The site address [https://slowfood-online-feb-17.herokuapp.com](https://slowfood
 * Dependencies:
   - acts as shoppingcart gem
   - devise gem
+  - To be able to run poltergeist you need PhantomJS, run:
```

Comment:
Add the strip gem and version and an explanation as to why we need to have it  in our gemfile. This will let other developers understand why we have included it and specified a version in our gemfile
Very long explanation for https://github.com/CraftAcademy/slow_food_online_feb_17/pull/24/files#r110140815